tests/seta/tests.py

```diff
@@ -0,0 +1,64 @@
+import datetime
+
+from django.utils import timezone
+
+from .models import (JobPriority,
+                     TaskRequest)
+
+
+class TaskRequestTests():
```

Comment:
Generally we just define tests as functions, that makes it easier to use fixtures. See, for example `tests/e2e/test_jobs_loaded.py`